ast/src/builder/core.rs

```diff
@@ -45,9 +45,8 @@ impl Repo {
 
         self.process_libraries(&mut graph, &filez)?;
         self.process_import_sections(&mut graph, &filez)?;
-        self.process_variables(&mut graph, &filez)?;
-        self.process_classes(&mut graph, &filez)?;
-        self.process_instances_and_traits(&mut graph, &filez)?;
+        self.process_independent_nodes(&mut graph, &filez)?;
```

Comment:
im not sure i understand what this changes. ideally we have each node type in its own function, so its easier to track progress and read